src/journal/journald-rate-limit.c

```diff
@@ -39,6 +39,7 @@ struct JournalRateLimitGroup {
         JournalRateLimit *parent;
 
         char *id;
+        usec_t interval;
```

Comment:
can you add a quick comment saying why we store this here? (i.e., that it's about expiry)